content/en/observability_pipelines/working_with_data.md

```diff
@@ -4,6 +4,9 @@ kind: Documentation
 aliases:
   - /integrations/observability_pipelines/working_with_data/
 further_reading:
+  - link: /observability_pipelines/installation/
```

Comment:
Quick question... Did you mean to link this to the Installation doc or the Getting Started guide?